client/src/components/EditableAttribute/EnumAttribute.js

```diff
@@ -2,24 +2,30 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
 
+const defaultDropdownOption = { label: '', value: '' };
+
 const EnumAttribute = ({
-  value = '',
+  value = defaultDropdownOption,
   valueOptions = [],
   attributeLabel = '',
   isDisabled = false,
   onChange,
 }) => {
-  const onValueChange = (option) => {
-    onChange(option, attributeLabel);
+  const onValueChange = (selectedOption) => {
+    onChange(selectedOption.value, attributeLabel);
+  };
+
+  const getOptionFromValue = (val) => {
+    const dropdownOption = valueOptions.find((option) => option.value === val);
+    if (dropdownOption) return dropdownOption;
+    return defaultDropdownOption;
   };
 
   return (
     <div>
       <p>{attributeLabel}</p>
       <Select
-        defaultValue={value}
-        value={value}
-        placeholder={value}
+        value={getOptionFromValue(value)}
```

Comment:
By default, `value` is `defaultDropdownOption`, which is an option object. But `getOptionFromValue` takes a string value and returns the corresponding option object. So default param for `value` should be `''`, since `getOptionFromValue` will return `defaultDropdownOption`?

Reply:
Yeah, good catch